sdk/eventhub/Azure.Messaging.EventHubs.Processor/README.md

```diff
@@ -171,6 +171,24 @@ private async Task ProcessUntilCanceled(CancellationToken cancellationToken)
 }
 ```
 
+### Authenticating with Azure.Identity
+
+The [Azure Identity library](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/identity/Azure.Identity/README.md) provides easy Azure Active Directory support for authentication.
```

Comment:
I'd strongly advise avoiding words like `easy`, as we'll have folks of various backgrounds and skill levels working with the client libraries and `Azure.Identity` may not feel easy or intuitive to some.

Maybe, something like:
```
The Azure Identity library offers support for using Azure Active Directory authentication with the Event Hubs clients.
```